### ISSUE TITLE
Luajit not support powerPC 64bit?

### ISSUE THREAD

**Issue by michaeldengchangshun** [hughperkins/distro#15]
when I installed on a powPC , it's always issue even I changed it to 2.0

**Comment by hughperkins** [comment]
Yeah.  its a known issue, I think, see the section 'embedding luajit' in http://luajit.org/install.html .  But I think it should work with `lua`, ie not `luajit`?
